doc: uv_close should be called after the exit callback (#1911) (#4164)
libuv/libuv: docs/src/guide/processes.rst

The exit callback will be invoked with the *exit status* and the type of *signal*
which caused the exit.

> Note that it is important not to call ``uv_close`` before the exit callback.

Reviewer: ```suggestion
Note that it is important **not** to call ``uv_close`` before the exit callback.
```
Just a suggestion but otherwise it's easy to overlook that not.

For processes started using libuv, you may use ``uv_process_kill`` instead,
which accepts the ``uv_process_t`` watcher as the first argument, rather than

> the pid. In this case, **remember to call** ``uv_close`` on the watcher after
> the execution of the exit callback.

Reviewer: ```suggestion
the pid. In this case, **remember to call** ``uv_close`` on the watcher _after_
the exit callback has been called.
```
Also just a suggestion but libuv's docs pretty much always say callbacks are called, not executed.
